examples/k8s/deployment.yaml

```diff
@@ -113,10 +116,10 @@ webhooks:
       port: 8443
       namespace: "devops"
   sideEffects: None
-  admissionReviewVersions: ["v1beta1"]
+  admissionReviewVersions: ["v1", "v1beta1"]
   failurePolicy: Ignore
-  name: v1beta1.kanopy-platform.github.io
-  reinvocationPolicy: IfNeeded
+  name: hedgetrimmer.kanopy-platform.github.io
```

Comment:
I wasn't sure why for the gateway-certificate-controller we had this as v1beta1, but I think it should deserve a proper name?

Reply:
`v1beta1.hedgetrimmer.kanopy-platform.github.io`